### DISCUSSION TITLE
trouble with the low-level backtest tutorial

### DISCUSSION THREAD

**Discussion by ViN807** [nautechsystems/nautilus_trader#1569] [Q&A] [answered]
I'm working through the low-level backtest tutorial and I'm having trouble with configuring the strategy, particularly the bar_type declaration. I think 'from nautilus_trader.model.data import BarType' should be included with the other import statements, but even when adding this I get the following error:

```
    bar_type=BarType("ETHUSDT.BINANCE-250-TICK-LAST-INTERNAL"),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "nautilus_trader\\model\\data.pyx", line 639, in nautilus_trader.model.data.BarType.__init__
TypeError: __init__() takes at least 2 positional arguments (1 given)
```
I can't determine if the issue is with parsing the argument to BarType or if its something else. Any help is appreciated.

**Comment by OnlyC** [comment, marked as answer]
You should use BarType.from_str("ETHUSDT.BINANCE-250-TICK-LAST-INTERNAL")

**Comment by cjdsellers** [comment]
Hi @ViN807

Thanks for helped there @OnlyC.

I also checked the entire codebase and thankfully we don't try to pass a string to the constructor anywhere else. You need to use the `from_str` method as OnlyC points out.
